packages/manager/modules/pci/src/projects/project/kubernetes/add/add.html

```diff
@@ -172,14 +173,17 @@ <h1 data-translate="kubernetes_add"></h1>
             <oui-field
                 data-size="m"
                 data-label="{{:: 'kubernetes_add_name' | translate }}"
+                data-error-messages="{pattern: ('kubernetes_add_name_input_help' | translate) }"
```

Comment:
**nitpick**: Since the error is for pattern, better to include `pattern` in the translation key.
```suggestion
                data-error-messages="{pattern: ('kubernetes_add_name_pattern_input_help' | translate) }"
```

Reply:
Done.